collectors/python.d.plugin/varnish/README.md

```diff
@@ -2,6 +2,14 @@
 
 Module uses the `varnishstat` command to provide varnish cache statistics.
 
+## Requirements
+
+User `netdata` should be added to the  `varnish` group. If you installed netdata **after** you installed varnish, then the user is already added to the proper group. Otherwise, please add it manually.
```

Comment:
```suggestion
User `netdata` should be added to the `varnish` group. If you installed Netdata **after** you installed varnish, then the user is already added to the proper group. Otherwise, please add it manually.
```